src/App.tsx

```diff
@@ -475,6 +499,12 @@ interface IOctant {
   octa: number
 }
 
+interface IRoad {
```

Comment:
This could be named differently. Maybe IQuadrant since it has 4 possibilities, when IOctant has 8 possibilities?